javascript/ql/lib/ext/react-relay-threat.model.yml

```
@@ -0,0 +1,15 @@
+extensions:
+  - addsTo:
+      pack: codeql/javascript-all
+      extensible: sourceModel
+    data:
+      - ["react-relay", "Member[useFragment].ReturnValue", "response"]
+      - ["react-relay", "Member[useLazyLoadQuery].ReturnValue", "response"]
+      - ["react-relay", "Member[usePreloadedQuery].ReturnValue", "response"]
+      - ["react-relay", "Member[useClientQuery].ReturnValue", "response"]
+      - ["react-relay", "Member[useRefetchableFragment].ReturnValue", "response"]
```

```suggestion
      - ["react-relay", "Member[useRefetchableFragment].ReturnValue.Member[0]", "response"]
```
Based on the code sample it seems the data is returned in the first array element:
```js
const [data, refetch] = useRefetchableFragment(
```